base/strings/string_util.h

```diff
@@ -47,7 +47,7 @@ inline int snprintf(char* buffer,
                     ...) {
   va_list arguments;
   va_start(arguments, format);
-  int result = vsnprintf(buffer, size, format, arguments);
+  int result = base::vsnprintf(buffer, size, format, arguments);
```

Comment:
Just use vsnprintf, it will resolve to base::vsnprintf and be consistent with the upstream code